hadoop-ozone/dist/src/main/smoketest/upgrade/snapshot.robot

```diff
@@ -0,0 +1,74 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Smoketest ozone cluster startup
+Library             OperatingSystem
+Library             BuiltIn
+Resource            ../commonlib.robot
+Resource            ../s3/commonawslib.robot
+Test Timeout        5 minutes
+
+*** Variables ***
+
+
+*** Test Cases ***
+Create snapshot
+    [Tags]     snapshot-support
+    ${output} =         Execute                ozone sh volume create vol
+                        Should not contain     ${output}       Failed
+    ${output} =         Execute                ozone sh bucket create /vol/bucket2
+                        Should not contain     ${output}       Failed
+    ${output} =         Execute                ozone sh snapshot create /vol/bucket2 snapshot1
+                        Should not contain     ${output}       Failed
+                        Execute and checkrc    echo "key created using Ozone Shell" > /tmp/sourcekey    0
+                        Execute                ozone sh key put /vol/bucket2/key1 /tmp/sourcekey
+                        Execute                ozone sh snapshot create /vol/bucket2 snapshot2
+
+Create snapshot in non-supported ozone version
+    [Tags]     snapshot-non-support
+    ${output} =         Execute and checkrc    ozone sh bucket create /vol/bucket2     255
```

Comment:
Is `bucket create` command enough to verify snapshot creation is not allowed?

Reply:
no.. I'm really sorry this test case wasn't completed yet. I need to add command to create snapshot after bucket creation.. thanks attila.